followthemoney/schema/CallForTenders.yaml

```diff
@@ -0,0 +1,122 @@
+CallForTenders:
+  label: "Call for tenders"
+  plural: "Calls for tenders"
+  description: >
+    A public appeal issued by an authority, possibly on behalf of another,
+    for buying a specific work, supply or service
+  extends:
+    - Thing
+    - Interval
+  matchable: false
+  required:
+    - title
+    - authority
+  featured:
+    - title
+    - authority
+  caption:
+    - title
+  properties:
+    callId:
+      label: "CfT unique id"
+      type: identifier
+    # What is "CA"? Contract award? Maybe it should then live there...
+    # cftCaUid:
+    #   label: "CfT CA unique id"
+    #   type: identifier
+    title:
+      label: "Title"
+    authority:
+      label: "Name of contracting authority"
+      reverse:
+        name: callForTenders
+        label: "Call For Tenders"
+      type: entity
+      range: LegalEntity
+    onBehalfOf:
+      label: "Published on behalf of"
+      reverse:
+        name: delegatedCallForTenders
+        label: "Delegated call for tenders"
+      type: entity
+      range: LegalEntity
+    publicationDate:
+      label: "Date of publication/invitation"
+      type: date
+    evaluationMechanism:
+      label: "Evaluation mechanism"
+    procurementType:
+      label: "Procurement type"
+    directive:
+      label: "Directive"
+    procedure:
+      label: "Procedure"
+    # What does this mean?
+    # involves:
+    #   label: "Call for tenders involves"
+    cpvCode:
+      label: "CPV codes"
+      description: "Common Procurement Vocabulary (CPV)"
+      type: identifier
+    # eAuctionsIncluded:
+    #   label: "Inclusion of e-Auctions"
```

Comment:
Ah, electronic auctions. This is a binary option. If yes then the CfT is a reverse auction. Worth keeping this field for statistical use. You could rename it to "reverse auction included" for example so that it's less vendor specific. Some information on "e-Auctions" in public bidding in Malta: [Reverse Bidding for Data Connectivity across the Public Service- A First for Malta](https://mita.gov.mt/2021/10/05/reverse-bidding-for-data-connectivity-across-the-public-service-a-first-for-malta/)